src/main/java/com/spartronics4915/frc2019/subsystems/Superstructure.java

```diff
@@ -463,7 +463,13 @@ public synchronized void setWantedState(WantedState wantedState)
 
     public synchronized void reverseDrivingDirection()
     {
-        mIsReversed = !mIsReversed;
+        mIsReversed = !mIsReversed
+        if(mIsReversed == true) {
+          dashboardPutBoolean("IsDriveReversed", true);
+        }
+        else if(mIsReversed == false) {
+          dashboardPutBoolean("IsDriveReversed", false);
```

Comment:
You should set `/SmartDashboard/Driver/Camera` to the camera name (something like `FrontFit` or `Test`, or whatever) and the cameras will switch for you.

We also need to discuss if we should switch cameras at all because of their flaky behavior when switching.